### PR TITLE
Themes/sko/breadcrumbs

### PR DESCRIPTION
**Pull request by Timo-Breumelhof**
Ok, this is the first SKO I do.
Checking if it's OK like this, to use as a basis for others.

### CODE REVIEW

File: content/tutorials/themes/theme-objects/breadcrumb/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+﻿---
+uid: breadcrumb
+locale: en
+title: Breadcrumb them object
```
Comment (david-poindexter) [inline content/tutorials/themes/theme-objects/breadcrumb/index.md]: `them` should be `theme`

File: content/tutorials/themes/theme-objects/breadcrumb/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+dnnversion: 09.02.00
+previous-topic: theme-objects
+next-topic: 
+related-topics: theme-objects,themes,create-theme
+links: 
+---
+
+# Breadcrumb Theme Object Introduction
+
+The Breadcrumb Them Object displays the path to the currently selected page in the menu structure like this:
```
Comment (david-poindexter) [inline content/tutorials/themes/theme-objects/breadcrumb/index.md]: @Timo-Breumelhof `Them` should be `Theme`

**Comment by Timo-Breumelhof** [comment]
Ok, thanks for the feedback. I first wanted to know if this was OK in itself, I'll make the needed changes

**Comment by Timo-Breumelhof** [comment]
@david-poindexter  BTW I did a restructuring of the main overview page, didn't want to mix those two :-)
https://github.com/DNNCommunity/DNNDocs/pull/471

**Pull request merged**